cdci_osa_plugin/osa_dataserve_dispatcher.py

```diff
@@ -126,7 +126,7 @@ def __init__(self,
         self.assume = assume
         self.inject = inject
 
-        self._test_products_with_astroquery = True
+        self._test_products_with_astroquery = False # TODO: this should be configurable
```

Comment:
What is the main of having this parameter?

Reply:
the main purpose? It decides if test_products are derived with astroquery - as used below.